tests/testthat/test-use-description.R

```diff
@@ -0,0 +1,81 @@
+context("use_description")
+
+test_that("build_description_list() defaults to values built into usethis", {
+  withr::with_options(
+    list(usethis.description = NULL, devtools.desc = NULL),
+    {
+      d <- build_description_list()
+      expect_identical(d$Version, "0.0.0.9000")
+      expect_match(d$Title, "What the Package Does")
+      expect_match(d$Description, "What the package does")
+      expect_match(d$License, "What license it uses")
+      expect_match(d$Encoding, "UTF-8")
+      expect_match(d$LazyData, "true")
+    }
+  )
+})
+
+test_that("build_description_list(): user's fields > usethis defaults", {
+  d <- build_description_list(
+    fields = list(Title = "aaa", URL = "https://www.r-project.org")
+  )
+  ## from user's fields
+  expect_identical(d$Title, "aaa")
```

Comment:
Why test two fields from each case? Isn't one enough?

Reply:
One is an overwrite, one is novel. But I did remove one test care re: default values.